backend/app/schemas/user.py

```diff
@@ -24,7 +24,7 @@ class RegisterUser(Auth):
     nickname: str | None = None
     email: str = Field(..., example='user@example.com')
 
-    @validator('email')
+    @field_validator('email')
     def email_validate(cls, v):
         try:
             validate_email(v, check_deliverability=False).email
```

Comment:
[https://docs.pydantic.dev/latest/api/networks/#pydantic.networks.EmailStr](https://docs.pydantic.dev/latest/api/networks/#pydantic.networks.EmailStr)

Reply:
I tried `EmailStr` in v1, but I forgot the reason why I didn't use it. I'll try again.

Reply:
updated.